src/content/ja/updates/2017/04/headless-chrome.md

```diff
@@ -0,0 +1,312 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ヘッドレス Chrome とはなにか、そしてその使い方を紹介します。
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
```

Comment:
We're missing some metadata that's [in the original](https://raw.githubusercontent.com/google/WebFundamentals/master/src/content/en/updates/2017/04/headless-chrome.md). They're required for all translations.
* wf_tags
* wf_featured_image
* wf_featured_snippet